sdk/metricsadvisor/Azure.AI.MetricsAdvisor/samples/README.md

```diff
@@ -24,24 +24,26 @@ Azure Cognitive Services Metrics Advisor is a cloud service that uses machine le
 |**Sample**|**Description**|
 |---|---|
 |[Data feed CRUD operations][metricsadv-sample1]|Create, get, update, list, and delete data feeds|
-|[Data feed ingestion operations][metricsadv-sample2]|Check and refresh a data feed's ingestion status|
-|[Anomaly detection configuration CRUD operations][metricsadv-sample3]|Create, get, update, list, and delete anomaly detection configurations|
-|[Hook CRUD operations][metricsadv-sample4]|Create, get, update, list, and delete hooks|
-|[Anomaly alert configuration CRUD operations][metricsadv-sample5]|Create, get, update, list, and delete anomaly alert configurations|
-|[Query triggered alerts][metricsadv-sample6]|Get the alerts triggered by the service|
-|[Query detected anomalies][metricsadv-sample7]|Get the anomalies detected by the service|
-|[Query incidents and their root causes][metricsadv-sample8]|Get the incidents created by the service and root cause analysis|
-|[Query time series information][metricsadv-sample9]|Get information about the time series monitored by the service|
-|[Feedback CRUD operations][metricsadv-sample10]|Create, get, and list feedback|
+|[Credential entity CRUD operations][metricsadv-sample2]|Create, get, update, list, and delete credential entities|
```

Comment:
Samples in here are not following our guideline of having a .md file that has more explanation and snippets of code.
do you have an issue to improve this in the future?

Reply:
Now we do (https://github.com/Azure/azure-sdk-for-net/issues/22556). Didn't know this was part of our guidelines.